### PR TITLE
feat: add css color markers project

### PR DESCRIPTION
**Pull request by scissorsneedfoodtoo**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by scissorsneedfoodtoo** [comment]
Here's the final version of the CSS Color Markers project I've been using to break down into steps. And here's a pen to make reviewing a bit easier: https://codepen.io/scissorsneedfoodtoo/pen/bGrVwoY?editors=1100

Could you take a look at this when you get a chance, @nhcarrigan and @ShaunSHamilton?

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍 

I cannot see anything missing in this, and it looks like a really decent project.

File: css-color-markers/styles.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  height: 25px;
+  display: inline-block;
+  vertical-align: top;
+  text-align: left;
+  padding-left: 10px;
+  font-size: 20px;
+  border-left: 10px double rgba(0, 0, 0, 0.75);
+}
+
+.marker > .sleeve {
```
Comment (nhcarrigan) [inline css-color-markers/styles.css:33]: A quick note that this will be the first time the `>` selector is introduced.

**Review by nhcarrigan — APPROVED** [review]
Looks good to me. Just a quick note to keep in mind with the steps.

File: css-color-markers/styles.css (hunk 1 above)
Comment (scissorsneedfoodtoo) [reply css-color-markers/styles.css:33]: Awesome, thanks for pointing this out.

Ahmad mentioned this to me in the past. I'm just getting around to that point in the steps, so I'll make sure to spend extra time on the `>` selector.

I'll also update the spreadsheet to reflect that now.

**Comment by ahmadabdolsaheb** [comment]
@scissorsneedfoodtoo looks solid,  the side color of some of the gradients could be adjusted. I was wondering if I should go in and change those if it is ok with you?

**Comment by scissorsneedfoodtoo** [comment]
Hey @ahmadabdolsaheb, sure, please go ahead and adjust those colors. I picked some of them pretty randomly, so I'd love to get your input.

**Comment by ahmadabdolsaheb** [comment]
Here are the modified gradients: https://codepen.io/abdolsa/pen/oNGbeEW

**Pull request merged**